lib/vector2.dart

```diff
@@ -0,0 +1,88 @@
+library flame.vector_math;
+
+export 'package:vector_math/vector_math_64.dart' show Vector2;
+
+import 'dart:math';
+import 'dart:ui';
+import 'package:vector_math/vector_math_64.dart';
+
+class Vector2Operations {
```

Comment:
Any consideration of why not using extension methods here?

Reply:
I wish, but since it uses factories in the constructors like this:
factory Vector2(double x, double y) => Vector2.zero()..setValues(x, y);
I haven't found a way to extend it nicely.
If you have any suggestions of how to do it I'm all ears.

Reply:
I will think about it but I think we can go with this for now... if it's impossible we can just live with the static funcs